app-frontend/src/app/pages/lab/analysis/analysis.js

```diff
@@ -28,7 +28,7 @@ class LabAnalysisController {
         this.tileServer = `${this.APP_CONFIG.tileServerLocation}`;
         this.showDiagram = true;
         this.analysisId = this.$state.params.analysisid;
-
+        this.backsplashTileMaxZoom = BUILDCONFIG.BACKSPLASH_TILE_MAX_ZOOM;
```

Comment:
Not sure there's a good reason to assign this instead of using the variable directly; I usually do this when it's a long name that's being used more than once for conciseness. Since this is a BUILDCONFIG variable, it gets substituted in at compile time.